File: SocialNetwork/SocialNetwork.Library/Models/Message.cs
using System;

namespace SocialNetwork.Library.Models
{
  public class Message
  {
    public DateTime CreatedOn { get; private set; }
    public string User { get; private set; }
    public string Text { get; private set; }

    public Message(DateTime createdOn, string user, string text)
    {
      CreatedOn = createdOn;
      User = user;
      Text = text;
    }
  }
}

File: SocialNetwork-Python/Message.py
__author__ = 'marcel'

class Message:
    " The message "

    def __init__(self, user, text, createdOn):
        self.user = user
        self.text = text
        self.createdOn = createdOn

    def __eq__(self, other):
        return self.user == other.user and self.text == other.text and self.createdOn == other.createdOn

    def __repr__(self):
        return self.user + " said " + self.text + " at " + str(self.createdOn)
